### DISCUSSION TITLE
Issue running pipeline using example data after installation

### DISCUSSION THREAD

**Discussion by rm1238** [aryam7/varCA#45] [Q&A]
Hello,

I recently downloaded VarCA to run on my ATAC data. Installation was apparently successful, but upon running "./run.bash &" from within the VarCA directory I get an exit (1) error. All dependencies were installed correctly, as I did not get any errors. Just wondering what could be triggering this error. 

I am running Python 3.10.5 on a MacBook Pro 2.4 GHz 8-Core Intel Core i9, 64GB RAM with OS X 12.4.

Thanks for your help,

Ricardo

![Screen Shot 2022-06-21 at 11 07 16](https://user-images.githubusercontent.com/49791648/174833771-17408441-0635-48a8-8d16-d65d07c7f68f.png)

**Comment by xuanxuan-Jin** [comment]
you should upload your log file when you run failed.

**Comment by rm1238** [comment]
loaded the log in another thread
